.gitignore

```diff
@@ -5,3 +5,6 @@ vendor/
 .php_cs.cache
 doc/.couscous
 .subsplit
+/config.yml
+/.buildpath
+/.project
```

Comment:
You should use a global gitignore for your personal tools: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c